js/nixos-site.js

```diff
@@ -1,3 +1,25 @@
+// FisherYates implementation
+$.fn.shuffleChildren = function(seed = Date.now()) {
+  var random = function(s) {
```

Comment:
Nitpick: could you rename this to `randomish`, to underline that the
distribution is not uniform, and to avoid shadowing the browser's `random`?

Reply:
Browser's random is `Math.random()`. Should we really rename it?